src/ocean/Ocean.C

```diff
@@ -52,38 +52,23 @@ extern "C" _SUBROUTINE_(set_seaice_parameters)(SeaIce::CommPars*);
 
 //=====================================================================
 // Constructor:
-Ocean::Ocean(RCP<Epetra_Comm> Comm, ParameterList oceanParamList)
+Ocean::Ocean(RCP<Epetra_Comm> Comm)
+    : Ocean(Comm, Teuchos::ParameterList())
+{}
+
+Ocean::Ocean(RCP<Epetra_Comm> Comm, Model::ParameterList paramListPtr)
+    : Ocean(Comm, paramListPtr == Teuchos::null ? Teuchos::ParameterList() : *paramListPtr)
+{}
+
+Ocean::Ocean(RCP<Epetra_Comm> Comm, const Teuchos::ParameterList& oceanParamList)
```

Comment:
You really want to pass a `Teuchos::RCP` here, see my comment for `setParameters`